src/aleph/vm/orchestrator/machine.py

```diff
@@ -0,0 +1,77 @@
+import json
+import re
+import subprocess
+from functools import lru_cache
+
+import psutil
+
+
+@lru_cache
+def get_hardware_info():
+    lshw = subprocess.Popen(["lshw", "-sanitize", "-json"], stdout=subprocess.PIPE, shell=False)
```

Comment:
A check in the settings that verifies that `lshw` is available on the machine would be a good addition.